proposals/tests/proposal_submission_tests.py

```diff
@@ -44,60 +47,53 @@ def setUp(self):
-            "user", "test@test.com", "secret", first_name="John", last_name="Doe"
-        )
-        self.supervisor = User.objects.create_user(
-            "supervisor", "test@test.com", "secret", first_name="Jane", last_name="Roe"
-        )
-
-        self.secretary.groups.add(
-            Group.objects.get(name=settings.GROUP_PRIMARY_SECRETARY)
-        )
-        self.secretary.groups.add(Group.objects.get(name=settings.GROUP_SECRETARY))
-
-        self.c1.groups.add(Group.objects.get(name=settings.GROUP_LINGUISTICS_CHAMBER))
-        self.c2.groups.add(Group.objects.get(name=settings.GROUP_LINGUISTICS_CHAMBER))
+        self.secretary = User.objects.get(username="secretary")
+        self.c1 = User.objects.get(username="c1")
+        self.c2 = User.objects.get(username="c2")
+        self.user = User.objects.get(username="user")
+        self.supervisor = User.objects.get(username="supervisor")
 
     def setup_proposal(self):
         """
-        Load our test proposals from a fixture, then add our user as
-        an applicant to each of them.
-
-        Please note, this currently uses a mish-mash of both fixtures
-        and programmatically created users and objects.
+        Load our test proposals from a fixture
         """
-        self.wmo = Wmo(
-            status=0,
-            metc="N",
-            is_medical="N",
+        self.proposal = Proposal.objects.get(
+            reference_number="24-009-01", title="Normal test proposal"
+        )
+        self.pre_assessment = Proposal.objects.get(
+            reference_number="24-010-01", title="Preassessment test proposal"
+        )
+        self.pre_approval = Proposal.objects.get(
+            reference_number="24-011-01", title="Preapproved test pdf"
         )
-        self.wmo.save()
-        self.proposal = Proposal.objects.get(pk=1)
-        self.proposal.applicants.add(self.user)
-        self.proposal.wmo = self.wmo
-        self.proposal.save()
-        self.pre_assessment = Proposal.objects.get(pk=2)
-        self.pre_assessment.applicants.add(self.user)
-        self.pre_assessment.save()
-        self.pre_approval = Proposal.objects.get(pk=3)
-        self.pre_approval.applicants.add(self.user)
-        self.pre_approval.save()
 
     def refresh(self):
         """Refresh objects from DB. This is sometimes necessary if you access
         attributes you previously read during the test and don't want to
         receive a cached value."""
         self.proposal.refresh_from_db()
+        self.pre_assessment.refresh_from_db()
+        self.pre_approval.refresh_from_db()
+
+    def add_supervisor_to_proposal(self):
+        self.proposal.relation = Relation.objects.get(
+            description_nl="als AIO / promovendus verbonden"
+        )
+        self.proposal.supervisor = self.supervisor
+        self.proposal.save()
+
+    def remove_study(self):
+        Study.objects.filter(proposal=f"{self.proposal.pk}").delete()
```

Comment:
Does `self.proposal.study_set.all().delete()` not work? As writting your version seems a bit omslachtig.

Also it removes all studies so best to call it `remove_studies()` (plural)